### PR TITLE
TMEDIA-758: Full Author Bio

### PR DESCRIPTION
**Pull request by nschubach**
## Description

Migrate Full Author Bio Block to V2

## Jira Ticket

- [TMEDIA-758](https://arcpublishing.atlassian.net/browse/TMEDIA-758)

## Acceptance Criteria

Full Author Bio block is migrated to the new architecture.

## Test Steps

1. Checkout branch `git checkout tmedia-758`
2. Checkout branch `git checkout tmedia-758` in `arc-themes-feature-pack`
3. Run fusion repo with linked blocks `npx fusion start -f -l`
4. Add block to the page and set page Global Content to `content-api` using `id`: `DLVY4FPTRRE2LM3IJXQF3V3B34`
5. Set page Global Content to `author-api` using `slug`: `tariq-alshwaiki`
6. Run Storybook via `npm run storybook`
7. Validate story

### After

image

## Dependencies or Side Effects

Feature Pack: https://github.com/WPMedia/arc-themes-feature-pack/pull/338
Need the global changes from: https://github.com/WPMedia/arc-themes-blocks/pull/1474

## Author Checklist

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

**Comment by ecarlisle** [comment]
@haganka: VQA question. This is the layout at 767px viewport width, just before the desktop layout kicks in. Just wanted to confirm it is per expectations:
image

File: blocks/full-author-bio-block/features/full-author-bio/_children/Presentation/index.jsx
Comment (matthewroach) [inline blocks/full-author-bio-block/features/full-author-bio/_children/Presentation/index.jsx]: We can remove the `rel` and `target` attributes as these are taken care of with the prop `opensInNewTab`

File: blocks/full-author-bio-block/package.json
Comment (matthewroach) [inline blocks/full-author-bio-block/package.json]:
```suggestion
	"description": "Full Author Bio Block",
```

**Review by matthewroach — CHANGES_REQUESTED** [review]
Have some concerns/questions around the design changes that I'll bring up in design sync today

File: blocks/full-author-bio-block/features/full-author-bio/_children/Presentation/index.jsx
Comment (nschubach) [reply blocks/full-author-bio-block/features/full-author-bio/_children/Presentation/index.jsx]: updated

**Comment by nschubach** [comment]
Design changes have been implemented

File: .storybook/themes/news.scss
Comment (matthewroach) [inline .storybook/themes/news.scss]: We should not be setting a max-width on the block even if `30rem` is pretty wide

File: .storybook/themes/news.scss
Comment (matthewroach) [inline .storybook/themes/news.scss]:
```suggestion
					font-weight: var(--heading-level-3-font-weight),
```

File: .storybook/themes/news.scss
Comment (matthewroach) [inline .storybook/themes/news.scss]:
```suggestion
					font-weight: var(--heading-level-6-font-weight),
```

File: .storybook/themes/news.scss
Comment (matthewroach) [inline .storybook/themes/news.scss]:
```suggestion
					font-weight: var(--heading-level-6-font-weight),
```

File: blocks/full-author-bio-block/features/full-author-bio/_children/Presentation/index.jsx
Comment (matthewroach) [inline blocks/full-author-bio-block/features/full-author-bio/_children/Presentation/index.jsx]: These phrases are not found with the changes in this pull requests - does this pull request depend on #1474 ?

**Review by matthewroach — CHANGES_REQUESTED** [review]

File: blocks/full-author-bio-block/features/full-author-bio/_children/Presentation/index.jsx
Comment (nschubach) [reply blocks/full-author-bio-block/features/full-author-bio/_children/Presentation/index.jsx]: Yeah:
image

File: .storybook/themes/news.scss
Comment (nschubach) [reply .storybook/themes/news.scss]:
```suggestion
```

**Comment by nschubach** [comment]
Original Chromatic build (re-running to test chromatic issue): https://www.chromatic.com/build?appId=5f91e4721ccaba0022a10782&number=3078

**Review by matthewroach — APPROVED** [review]

File: locale/de.json
Comment (JackHowa) [inline locale/de.json]: can you delete this from lokalise as well? It looks like I can see it in lokalise. want to make sure we're accounting for changes so that the global phrases in lokalise can have this info. Let me know if you want help working with lokalise

Screen Shot 2022-08-30 at 15 07 52

**Review by JackHowa — COMMENTED** [review]
Had a translation question. I may have missed something from a previous discussion

File: blocks/full-author-bio-block/features/full-author-bio/_children/Presentation/index.test.jsx
Comment (JackHowa) [inline blocks/full-author-bio-block/features/full-author-bio/_children/Presentation/index.test.jsx:8]: nice use of testid

**Review by JackHowa — CHANGES_REQUESTED** [review]
Can you upload your en.json to lokalise. it looks like there hasn't been an upload with full author bio. Not sure if you're collaborating with the short author bio on this

**Comment by JackHowa** [comment]
pr is waiting on translations from the global phrases block in other short author bio pr

File: locale/de.json
Comment (JackHowa) [reply locale/de.json]: I guess migrate them

Comment (JackHowa) [reply locale/de.json]: we want to make sure we keep the translations

**Review by JackHowa — APPROVED** [review]
Good to merge once the translations pr is merged in here and -connect has been updated

File: blocks/full-author-bio-block/features/full-author-bio/default.jsx
Comment (nschubach) [inline blocks/full-author-bio-block/features/full-author-bio/default.jsx]:
```suggestion
	const { locale } = getProperties(arcSite);
```

**Review by JackHowa — DISMISSED** [review]
Looks like some merge conflicts. if this is dqa approved, happy to re-review

**Review by matthewroach — APPROVED** [review]

**Pull request merged**
